ui/apps/platform/cypress/helpers/wizard.ts

```diff
@@ -13,3 +13,11 @@ export function goToWizardStep(step: number | string) {
 export function getWizardStepTitle(title: string) {
     return cy.get('.pf-v5-c-wizard [data-ouia-component-type="PF5/Title"]').contains(title);
 }
+
+export function navigateWizardForward() {
```

Comment:
Although these names are intuitive, what are pro and con for reader:
* `Forward` versus `Next`
* `Backward` versus `Back`